Assets/SEE/Game/LabelAttributes.cs

```diff
@@ -24,13 +34,13 @@ public class LabelAttributes
         /// The distance between the top of the node and its label.
         /// </summary>
         [Tooltip("The distance between the top of the node and its label.")]
-        public float Distance = 0.2f;
+        public float Distance = DEFAULT_DISTANCE;
```

Comment:
Names must be in CamelCase.
Private fields must start with a lower-case letter, protected, internal and public fields with upper-case letter. This holds for constant and readonly declarations, too.

Reply:
Fixed in abf328e4b12dd2f3276416e37bbe466f2e306a5b.